AdrianaCano/SpringTest/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
What's the reason of this file if it's empty?

Reply:
That is because when we initialize a project, it is black by default and there we can configure some properties like the port where we want to run the project. I didn't add anything because I run the project on port 8080 and this is a default configuration. Let me know if I am mistaken, I am open to correction.